src/gui/toolbarMenubar/ToolMenuHandler.cpp

```diff
@@ -478,7 +478,11 @@ void ToolMenuHandler::initToolItems()
 	fontButton = new FontButton(listener, gui, "SELECT_FONT", ACTION_FONT_BUTTON_CHANGED, _("Select Font"));
 	addToolItem(fontButton);
 
-	ADD_CUSTOM_ITEM_TGL("RECSTOP", ACTION_RECSTOP, GROUP_REC, false, "rec", _("Rec / Stop"));
+	ADD_CUSTOM_ITEM_TGL("AUDIO_RECORDING", ACTION_AUDIO_RECORD, GROUP_AUDIO, false, "audio-record", _("Record Audio / Stop Recording"));
```

Comment:
If you change something like this, we need backward compatibility.... (not checked yet)

Reply:
Where and how do I add this?

Reply:
ToolMenuHandler::load, there is already such a backward fix:

```
				// recognize previous name, V1.07 (Jan 2019) and earlier.
				if (name == "TWO_PAGES")
				{
					name = "PAIRED_PAGES";
				}
```